src/Database/LibSQLConnection.php

```diff
@@ -30,33 +31,6 @@ public function sync(): void
         $this->db->sync();
     }
 
-    public function getDb(): LibSQL
```

Comment:
@darkterminal As we were talking https://github.com/tursodatabase/turso-driver-laravel/pull/5#discussion_r1669057147

I don't think it's good idea to have this methods, because they are ambiguous.

Reply:
Would it be better if this were reserved for internal derivatives only?

Reply:
You can already access `$this->db`, and even tho, since it's actually public it's not reserved for internal usage only, maybe it would be better to set it to protected.

Reply:
Make sense... the property are need to be protected.